src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -7472,6 +7472,34 @@ static void Main()
 
             CleanupAllGeneratedFiles(src.Path);
         }
+
+        [Fact]
+        public void ParseAnalyzerDependencies()
+        {
+            var parsedArgs = CSharpCommandLineParser.Default.Parse(new string[] { @"/analyzerdependency:foo.dll", "a.cs" }, _baseDirectory);
+            parsedArgs.Errors.Verify();
+            Assert.Equal(1, parsedArgs.AnalyzerDependencies.Length);
+            Assert.Equal("foo.dll", parsedArgs.AnalyzerDependencies[0].FilePath);
+
+            parsedArgs = CSharpCommandLineParser.Default.Parse(new string[] { "/analyzerdependency:\"foo.dll\"", "a.cs" }, _baseDirectory);
+            parsedArgs.Errors.Verify();
+            Assert.Equal(1, parsedArgs.AnalyzerDependencies.Length);
+            Assert.Equal("foo.dll", parsedArgs.AnalyzerDependencies[0].FilePath);
+
+            parsedArgs = CSharpCommandLineParser.Default.Parse(new string[] { @"/analyzerdependency:foo.dll;bar.dll", "a.cs" }, _baseDirectory);
+            parsedArgs.Errors.Verify();
+            Assert.Equal(2, parsedArgs.AnalyzerDependencies.Length);
+            Assert.Equal("foo.dll", parsedArgs.AnalyzerDependencies[0].FilePath);
+            Assert.Equal("bar.dll", parsedArgs.AnalyzerDependencies[1].FilePath);
```

Comment:
Add a test where the flag is used multiple times i.e. @"/analyzerdependency:a.dll;b.dll", @"/analyzerdependency:a1.dll;b1.dll" (avoid foo / bar if possible)